src/vivarium_public_health/risks/distributions.py

```diff
@@ -79,23 +79,23 @@ def initialization_requirements(self) -> Dict[str, List[str]]:
     def __init__(self, risk, weights, mean, sd):
         super().__init__()
         self.risk = EntityString(risk)
-        self.weights = weights
+        self.raw_weights = weights
         self.mean = mean
         self.standard_deviation = sd
         self._propensity = f"ensemble_propensity_{self.risk}"
 
     def setup(self, builder: Builder) -> None:
         self.randomness = builder.randomness.get_stream(self._propensity)
-        self._weights, self._parameters = self.get_parameters(builder)
+        self.input_weights, self._parameters = self.get_parameters(builder)
 
     ##########################
     # Initialization methods #
     ##########################
 
     def create_lookup_tables(self, builder: Builder) -> None:
```

Comment:
This might not be the best place for this comment, but for consistency in naming, should this method be called `build_lookup_tables` instead?

Reply:
Yea I think we should change that.